models/intel/index.md

```diff
@@ -96,7 +96,8 @@ detect areas with complex shape (for example, free space on the road).
 |------------------------------------------------------------------------------------------------------------------------------------------------|--------------------- |----------- |
 | [road-segmentation-adas-0001](./road-segmentation-adas-0001/description/road-segmentation-adas-0001.md)                                        | 4.770                | 0.184      |
 | [semantic-segmentation-adas-0001](./semantic-segmentation-adas-0001/description/semantic-segmentation-adas-0001.md)                            | 58.572               | 6.686      |
-
+| [icnet-onnx-0001](./icnet-onnx-0001/description/icnet-onnx-0001.md)                            | -               | -      |
+| [unet-onnx-0001](./unet-onnx-0001/description/unet-onnx-0001.md)                            | -              | -      |
```

Comment:
Why is there no complexity/size listed?

Reply:
there will be no info on complexity/size at the moment as model analyzer not ready yet.

Reply:
@vshampor is not model name is icnet-vamvid-onnx-0001 and unet-camvid-onnx-0001 ?